Makefile

```diff
@@ -133,8 +133,8 @@ test/arg.beam: test/should_fail/arg.erl
 .PHONY: build_test_data
 test_data_erls = $(wildcard test/known_problems/**/*.erl test/should_fail/*.erl test/should_pass/*.erl)
 build_test_data:
-	mkdir -p "test_data"
-	erlc $(TEST_ERLC_OPTS) -o test_data $(test_data_erls)
+	@mkdir -p "test_data"
+	@erlc $(TEST_ERLC_OPTS) -o test_data $(test_data_erls)
```

Comment:
Hehe, I've always been annoyed by build tools that don't show they are doing, i.e. don't print the commands that run (like CMake or Rebar, they hide everything by default). In make you at least see the commands by default. When the commands are printed, you can understand what happens and you can easily repeat the same commands manually.

I'm not going to argue, but I am going to unassign myself as a reviewer of this PR. :)

Reply:
In general I agree, I don't like opaque complexity either. With this number of repetitions, though, I could tell by heart what it's doing ;) I don't really need to see again and again the parts that don't change, I'm more interested in the output which does change. Anyway, this bit is a marginal detail.

Reply:
If the files don't change, they targets shouldn't need to be rebuilt. Some dependency declaration missing here seems to be the cause of repetition.

Reply:
I agree that the Makefile can probably be tweaked so that tests are built only when they actually change. I would consider that a better option than just silencing the commands.